### PR TITLE
Test extension on Travis

### PR DESCRIPTION
**Pull request by Komzpa**

### CODE REVIEW

**Comment by strk** [comment]
Note that `make installcheck` already does the extension test, no need to override!

**Comment by strk** [comment]
Rather, it would help if travis did run _upgrade_ tests (currently only run by other bots which not many people like to listen to...)

File: ci/travis/run_tests.sh [hunk 1]
```diff
@@ -16,4 +16,4 @@ export CUNIT_VALGRIND_FLAGS="--leak-check=full --error-exitcode=1"
 
 # Standard build
 ./configure CFLAGS="${CFLAGS_STD}" LDFLAGS="${LDFLAGS_STD}"
-bash ./ci/travis/logbt -- make -j check RUNTESTFLAGS="--verbose --extension"
+bash ./ci/travis/logbt -- make -j installcheck RUNTESTFLAGS="--verbose"
```
Comment (strk) [inline ci/travis/run_tests.sh]: We want _both_ `make check` and `make installcheck`. The first checks the installation via manual scripts, the second checks via extension

Comment (strk) [reply ci/travis/run_tests.sh]: Of course the `installcheck` must be done _after_ `make install`

**Comment by Komzpa** [comment]
we have drone now.

**Pull request closed without merging**